app/collectors/reservation.scala

```diff
@@ -0,0 +1,56 @@
+package collectors
+
+import agent._
+import com.amazonaws.services.ec2.AmazonEC2Client
+import com.amazonaws.services.ec2.model.{DescribeReservedInstancesRequest, ReservedInstances}
+import controllers.routes
+import org.joda.time.{DateTime, Duration}
+import play.api.mvc.Call
+import utils.Logging
+
+import collection.JavaConverters._
+import scala.util.Try
+
+object ReservationCollectorSet extends CollectorSet[Reservation](ResourceType("reservations", Duration.standardMinutes(15L))) {
```

Comment:
I think I prefer the singular here, but having looked there is no consistency across the code base :(